examples/fluids/qfunctions/dirichlet_boundary.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and other CEED contributors.
+// All Rights Reserved. See the top-level LICENSE and NOTICE files for details.
+//
+// SPDX-License-Identifier: BSD-2-Clause
+//
+// This file is part of CEED:  http://github.com/ceed
+
+#ifndef dirichlet_boundary_h
+#define dirichlet_boundary_h
+
+#include <ceed.h>
+
+CEED_QFUNCTION(SetupDirichletBC)(void *ctx, CeedInt Q,
+                                 const CeedScalar *const *in,
+                                 CeedScalar *const *out) {
+  // Inputs
+  const CeedScalar (*coords)[CEED_Q_VLA] = (const CeedScalar(*)[CEED_Q_VLA])in[0];
+  const CeedScalar (*multiplicity)       = (const CeedScalar(*))in[1];
+
+  // Outputs
+  CeedScalar (*coords_stored)[CEED_Q_VLA] = (CeedScalar(*)[CEED_Q_VLA])out[0];
+  CeedScalar (*scale_stored)              = (CeedScalar(*))out[1];
+
+  CeedPragmaSIMD
+  for(CeedInt i=0; i<Q; i++) {
+    for (int j=0; j<3; j++) coords_stored[j][i] = coords[j][i];
+    scale_stored[i] = 1.0 / multiplicity[i];
```

Comment:
Minor - I don't think it has anything to do with correctness, but `multiplicity` has 5 components here

Reply:
Actually, I'm not sure if it would cause an issue with the code generation or not

Reply:
Hmmm. Didn't realize that. 

Is there a circumstance in which the five entries of multiplicity would be different? I think that would require that certain components not be shared at nodes, but I can't think of a circumstance where that's true other than (maybe?) mixed FEM.

Reply:
all 5 fields should be the same in this case